src/main/webapp/app/shared/util/common-utils.ts

```diff
@@ -9,3 +9,31 @@ export const getFirstLettersFromString = (stringText: string): string => {
   else finalString = splitString[0].slice(0, 2);
   return finalString.toUpperCase();
 };
+
+export const parseBool = str => {
+  if (str.length === null) {
```

Comment:
this is confusing it should be !str && str.length === 0l

Reply:
done